README.md

```diff
@@ -54,7 +54,7 @@ fastify.register(fastifyRequestContextPlugin, {
 This plugin accepts options `hook` and `defaultStoreValues`. 
 
 * `hook` allows you to specify to which lifecycle hook should request context initialization be bound. Note that you need to initialize it on the earliest lifecycle stage that you intend to use it in, or earlier. Default value is `onRequest`.
-* `defaultStoreValues` sets initial values for the store (that can be later overwritten during request execution if needed). This is an optional parameter.
+* `defaultStoreValues` / `defaultStoreValues(req: FastifyRequest)` sets initial values for the store (that can be later overwritten during request execution if needed). Can be set to either an object or a function that returns an object. The function will be sent the request object for the new context. This is an optional parameter.
```

Comment:
can you make it more explicit that it can be either a Record or a function?

Reply:
Any suggestion? Not sure how much clearer I can make it

Reply:
reread it, it's clear enough